PRIVACY.md

```diff
@@ -0,0 +1,21 @@
+# Semgrep Privacy Policy
+
+Semgrep collects usage data to help us improve the product. We send two types of data to r2c servers: scan data and findings data.
+
+## Scan data
+
+Scan data provide information on the environment and performance of Semgrep. They power dashboards, identify anomalies with the product, and are needed for billing. The classes of data included are:
+- Project identity (e.g. name, URL)
+- Scan environment (e.g. CI provider, OS)
+- Author identity (e.g. committer email)
+- Commit metadata (e.g. commit hash)
+- Scan metadata, including type of scan and scan parameters (e.g. files or branches to ignore)
```

Comment:
What are some examples of this data? Is file information needed? See later comments on not including file path in Findings data.

Reply:
Would this data include timing information?

Reply:
yes aggregate timing and performance data is collected so we can debug and improve semgrep performance proactively

Reply:
Bence posted some examples he pulled from the db in [this issue](https://github.com/returntocorp/semgrep-app/issues/502).